src/Illuminate/Foundation/Configuration/ApplicationBuilder.php

```diff
@@ -274,7 +283,7 @@ public function withCommands(array $commands = [])
     protected function withCommandRouting(array $paths)
     {
         $this->app->afterResolving(ConsoleKernel::class, function ($kernel) use ($paths) {
-            $this->app->booted(fn () => $kernel->addCommandRoutePaths($paths));
+            $kernel->setCommandRoutePaths($paths);
```

Comment:
Please undo these changes from #50742.

Reply:
This is my first contribution in laravel if you can share more information how can which method should I undo that will help me to do this